src/app/external-cluster-wizard/steps/external-cluster/provider/gke/component.ts

```diff
@@ -44,10 +44,13 @@ import {MachineDeploymentSpec} from '@app/shared/entity/machine-deployment';
 import {MatCheckboxChange} from '@angular/material/checkbox';
 import {GCPDiskType, GCPMachineSize} from '@app/shared/entity/provider/gcp';
 import {NameGeneratorService} from '@app/core/services/name-generator';
+import {MasterVersion} from '@app/shared/entity/cluster';
 
 enum Controls {
   Name = 'name',
   Zone = 'zone',
+  KubernetesVersionMode = 'kubernetesVersionMode',
```

Comment:
move these below for better readability lol. you can ignore it if you want too ;)
```
-
--
---
-----
-------
```